modules/Display.js

```diff
@@ -1,6 +1,4 @@
-Processes.listOfDevices['display'] = {
-    name: "Display",
```

Comment:
They're in initd in order to make the file relevant to the assignment:
"Implement initd to start all the device drivers and the CLI process."
The start device drivers part, specifically. There wasn't really much to start.